worker/src/RTC/Transport.cpp

```diff
@@ -2900,8 +3029,10 @@ namespace RTC
 
 		MS_DEBUG_DEV("outgoing available bitrate:%" PRIu32, bitrates.availableBitrate);
 
+#ifndef USE_RTC_SENDER_BANDWIDTH_ESTIMATOR
 		DistributeAvailableOutgoingBitrate();
 		ComputeOutgoingDesiredBitrate();
+#endif
```

Comment:
So we are no longer distributing available bitrate since `USE_RTC_SENDER_BANDWIDTH_ESTIMATOR` is not defined. Why?

Reply:
if `USE_RTC_SENDER_BANDWIDTH_ESTIMATOR` **is not** defined we distribute TCC calculated BWE, otherwise we distribute SenderBandwidthEstimator BWE.

Reply:
Didn't realize that it's `#ifndef USE_RTC_SENDER_BANDWIDTH_ESTIMATOR`, it's ok.